beacon-chain/rpc/eth/light-client/handlers.go

```diff
@@ -268,7 +272,41 @@ func (s *Server) GetLightClientFinalityUpdate(w http.ResponseWriter, req *http.R
 			finalizedBlock = nil
 		}
 	}
+	// Check if the client requests SSZ format
+	if httputil.RespondWithSsz(req) {
+		update, err := lightclient.NewLightClientFinalityUpdateFromBeaconState(ctx,
+			state,
+			block,
+			attestedState,
+			finalizedBlock)
+		if err != nil {
+			httputil.HandleError(w, "could not get light client finality update: "+err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+		Pubkeys := make([][]byte, 512)
+		for i := range Pubkeys {
+			Pubkeys[i] = make([]byte, 48)
+		}
+		NextSyncCommitteeBranch := make([][]byte, 5)
+		for i := range NextSyncCommitteeBranch {
+			NextSyncCommitteeBranch[i] = make([]byte, 32)
+		}
+		update.NextSyncCommitteeBranch = NextSyncCommitteeBranch
+		update.NextSyncCommittee = &eth.SyncCommittee{
+			Pubkeys:         Pubkeys,
+			AggregatePubkey: make([]byte, 48),
+		}
```

Comment:
This is very wrong. You are replacing the correct data coming from `NewLightClientFinalityUpdateFromBeaconState` with empty values. This is what will be returned to the caller in production